src/lib/components/Header/Nav/NavItem.svelte

```diff
@@ -1,108 +1,21 @@
 <script lang="ts">
-  import type { NavItemProps, NavItemType } from "./NavItemType";
+  import type { NavItemProps, NavLinkType } from "./types";
 
-  import classNames from "$lib/util/classNames";
-  import chunk from "$lib/util/chunk";
-  import Link from "$lib/components/Link";
+  import NavLink from "./NavLink.svelte";
+  import NavMenu from "./NavMenu.svelte";
 
   type $$Props = NavItemProps;
 
   export let id = "";
-  export let link = "";
-  export let isCurrent = false;
-  export let megaMenuColumns = 0;
-  export let children: NavItemType[] = [];
-
-  let isParent = children.length > 0;
-  let isExpanded = false;
-
-  $: buttonClassNames = classNames(
-    "usa-accordion__button usa-nav__link",
-    isCurrent && "usa-current"
-  );
-  $: navLinkClassNames = classNames("usa-nav-link", isCurrent && "usa-current");
-
-  // Divide nav items into columns.
-  // TODO: We may want to let the CMS determine the position of each item in the future.
-  let megaMenu: NavItemType[][] = [];
-  if (megaMenuColumns > 0) {
-    const columnLength = Math.ceil(children.length / megaMenuColumns);
-    megaMenu = chunk(children, columnLength);
-  }
-
-  const toggleSubMenu = (show?: boolean) => {
-    if (typeof show !== "undefined") isExpanded = show;
-    else isExpanded = !isExpanded;
-  };
-
-  // By default focusout on the container will trigger for all its children, but using 'self'
-  //   doesn't resolve the issue since the container element isn't the one with the focus.
-  // This workaround will ignore the event if the new focused element is a child of the container.
-  // Based on this REPL: https://svelte.dev/repl/4c5dfd34cc634774bd242725f0fc2dab?version=3.46.4
-  const handleDropdownFocusLoss = ({
-    relatedTarget,
-    currentTarget,
-  }: FocusEvent & { currentTarget: EventTarget & HTMLDivElement }) => {
-    if (relatedTarget instanceof HTMLElement && currentTarget?.contains(relatedTarget)) return;
-    toggleSubMenu(false);
-  };
-
-  // We can't use on:click since it only triggers if the mousedown and mouseup events occur on the
-  //   same target, and on mobile the collapse of an accordion when it loses focus happens on
-  //   mousedown, sometimes shifting the elements so that mouseup misses the target.
-  // Instead, we'll use both mousedown and keydown to ensure everything functions accessibly on
-  //   both desktop and mobile screen sizes.
-  const handleDropdownKeyDown = (event: KeyboardEvent) => {
-    if (event.key === "Enter" || event.key === " ") toggleSubMenu();
-  };
+  export let link: string | undefined = undefined;
+  export let current = false;
```

Comment:
This needs to be changed in the story:https://github.com/la-ldaf/ldaf-site/blob/784dfc99e10cc383d6f4e39a03e1dbd7f8f6a589/src/stories/Header.stories.ts#L32

And the unit test:
https://github.com/la-ldaf/ldaf-site/blob/784dfc99e10cc383d6f4e39a03e1dbd7f8f6a589/src/lib/components/Header/Nav/NavItem.test.ts#L28

Honestly I probably should have checked that the element was getting the expected class name when it was set to `current`, so I'll probably handle that as a follow-up in my PR.